RadixWallet/Clients/SecureStorageClient/SecureStorageClient+Live.swift

```diff
@@ -77,29 +77,20 @@ extension SecureStorageClient: DependencyKey {
 
 		@Sendable func saveProfile(
 			snapshotData data: Data,
-			key: KeychainClient.Key,
-			iCloudSyncEnabled: Bool
+			key: KeychainClient.Key
 		) throws {
 			try keychainClient.setDataWithoutAuth(
 				data,
 				forKey: key,
 				attributes: .init(
-					iCloudSyncEnabled: iCloudSyncEnabled,
+					iCloudSyncEnabled: false,
```

Comment:
I think that profile header list should also not be synced with Cloud. In SelectBackup, the profile header list is read, and then the id is used to load from keychain. Since profile is not synced anymore with keychain cloud, it is possible that the header will contain the id of profile missing from the keychain. Based on CloudBackupClient code, the migration will fail with `ProfileMissingFromKeychainError`.

Reply:
alright

Reply:
But at the same time, we don't turn off syncing for migrated profiles, only for the currently active profile.

In your scenario: 
- we turn off syncing for profile X, the active one on phone A
- stop syncing the header list on phone A
- phone B asks for the list, it will include X, but X is not in iCloud keychain

Depending on how that list is maintained, it seems like we should keep it in sync?